### ISSUE TITLE
v1.5.1, peer dependency "sinon" is missing

### ISSUE THREAD

**Issue by thomasweitzel** [Polymer/polymer-cli#854]
### Description

Update to version 1.5.1 leads to error `npm ERR! peer dep missing: sinon@^1.4.0 || ^2.1.0 || ^3.0.0, required by sinon-chai@2.13.0` (`npm WARN sinon-chai@2.13.0 requires a peer of sinon@^1.4.0 || ^2.1.0 || ^3.0.0 but none was installed.` is shown during update).

### Versions & Environment

- Polymer CLI: 1.5.1
- node: 6.11.2
- Operating System: Ubuntu 14.04 (4.4.0-92-generic)

#### Steps to Reproduce

1. Update from previous version with `npm -g update`
2. List installed packages with `npm -g list --depth=0`

#### Expected Results

```
/home/test/.npm-global/lib
├── bower@1.8.0
├── npm@5.3.0
└── polymer-cli@1.5.1
```

#### Actual Results

```
/home/test/.npm-global/lib
├── bower@1.8.0
├── npm@5.3.0
└── polymer-cli@1.5.1

npm ERR! peer dep missing: sinon@^1.4.0 || ^2.1.0 || ^3.0.0, required by sinon-chai@2.13.0
```
#### Manual fix
Install missing dependency with `npm -g install sinon`.

**Comment by usergenic** [comment]
Thanks for filling this issue.  Because sinon was not web-ready we switched to a clone of the sinonjs Bower package for npm but while this satisfies the runtime dependency it doesn't satisfy npm package dependency check.  Probably will add sinon npm dependency for now to suppress this warning/error at install.

**Comment by usergenic** [comment]
This was resolved a few weeks ago, I believe with the 1.5.2 release (or shortly thereafter.)

Resolution was addition of sinon as a explicit dependency even though the actual code used comes from `@polymer/sinonjs` which is a web-ready version cloned from the bower-based sinonjs forked from forked from `https://github.com/blittle/sinon.js`
